Gopkg.toml

```diff
@@ -57,3 +57,7 @@
 [[constraint]]
   name = "gopkg.in/AlecAivazis/survey.v1"
   version = "1.6.1"
+
+[[constraint]]
+  name = "github.com/cloudevents/sdk-go"
+  revision = "feec6e002535db7561a364f82f67d2c9ebe4596a"
```

Comment:
nit: it would be more visible if you used the 0.2.0 release tag - https://github.com/cloudevents/sdk-go/releases/tag/0.2.0 - but clearly not a big issue.

Reply:
It's definitely more clear, fixing.